### DISCUSSION TITLE
Don't get response from whisper model

### DISCUSSION THREAD

**Discussion by kolobott** [deepgram/community#295] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

Hello,
Last week I noticed that I don't get response from whisper model and it can process 1 hour video more than 5 hours or even just got stuck. When do you resolve this problem?

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @kolobott, last week we made some infrastructure adjustments that led to some increased latency for Whisper. We've since made further improvements, so you should observe more speed and stability with receiving responses going forward.

If you notice a request in Pending (i.e. "stuck" as you mentioned) for more than a few hours, you can assume that you should retry the request. We are working on better messaging and error handling in these cases.

Note that Whisper is inherently slower and less scalable than Deepgram's own models. I highly encourage you to try our [Nova](https://deepgram.com/learn/nova-speech-to-text-whisper-api) model if you haven't already (parameter `model=nova` instead of `model=whisper`). It offers the best combination of speed, accuracy, and cost. Are there any specific features that are leading you to use Whisper instead of Nova?

**Reply by aqasemi** [reply]
Nova is quite good actually, but the problem is that it only supports a limited number of languages. Would be great if you could support more languages, like Arabic, Hebrew, and commonly used languages. Good work nonetheless. Thank you!
